guardian/newsletters-nx
# Updated script to create .env.local rather than simply replacing one which already exists

Fixes error in setup script



## What does this change?

The set-up scrip should now create, rather than just replace the `.env.local` config file. 



## How to test

- Back-up your `.env.loca`l from the newsletters-api project (if you have one)
- Run the script `./script/setup.sh`
- Check that the defauls have been copied in place of the existingvalues
- Delete the `.env.local`
- Run the script again
- Check it has been recreated

The script output shoud indocate whether the script has been created or replaced, and there is a  safetly check to have the user confirm the replace where a config file already exists.
